Sources/Layout/AnnounceMessageSizeCalculator.swift

```diff
@@ -7,6 +7,7 @@
 //
 
 import Foundation
+import UIKit
```

Comment:
From some version of Swift Package Manager, packages are cannot be compiled without declaring the correct imported modules.

In this file `UIFont`, `UIEdgeInsets` and other `UI`-prefixed class are used.